src/NUnitConsole/nunit3-console/Options.cs

```diff
@@ -158,13 +158,7 @@
 using NDesk.Options;
 #endif
 
-#if NUNIT_CONSOLE || NUNITLITE || NUNIT_ENGINE
```

Comment:
This is the only real change in this file. I edited in VSCode, so it dropped the trailing spaces on save. Sorry.

Reply:
Sometimes I wonder how much worse our lives would really be if somebody submitted a whitespace consistency PR so we never had to deal with this again? 🤔